src/App.js

```diff
@@ -3,15 +3,59 @@ import Header from './Header.js'
 import Footer from './Footer.js'
 import Main from './Main.js'
 import data from './data.json';
+import Modal from 'react-bootstrap/Modal';
 
 import './App.css';
 
 class App extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      show: false,
+      animal: {
+        title: 'proof of life'
+      }
+    }
+  }
+
+  handleShowModal = (animal) => {
+    this.setState({
+      show: true,
+      animal: animal
+    })
+  }
+
+  handleCloseModal = () => {
+    this.setState({
+      show: false
+    })
+  }
+
   render() {
+    console.log(this.state)
     return (
       <>
         <Header />
-        <Main data={data}/>
+
+        <Modal show={this.state.show} onHide={this.handleCloseModal}>
+          <Modal.Header closeButton>
+            <Modal.Title>{this.state.animal.title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <img 
+              width = '100%'
+              src={this.state.animal.image_url}
+              alt={this.state.animal.description}
+              title={this.state.animal.title}
+            />
+            {this.state.animal.description}
+          </Modal.Body>
+        </Modal>
+
```

Comment:
- once you have imported `SelectedBeast`, you can use the component here. 
- The component will take in some props, most likely some of the modal methods to hid and show and perhaps some state... 

Grab a TA so they can help ya move some code around if you get stuck 👍  I hope this helps :)